clients/feeder/feeder.go

```diff
@@ -262,14 +273,23 @@ func (c *Client) get(ctx context.Context, queryURL string) (io.ReadCloser, error
 				res.Body.Close()
 			}
 
+			c.curTimeout++
+			if c.curTimeout >= len(c.timeouts) {
+				c.curTimeout = len(c.timeouts) - 1
+			}
+
 			if wait < c.minWait {
 				wait = c.minWait
 			}
 			wait = c.backoff(wait)
 			if wait > c.maxWait {
 				wait = c.maxWait
 			}
-			c.log.Debugw("Failed query to feeder, retrying...", "req", req.URL.String(), "retryAfter", wait.String(), "err", err)
+			c.log.Debugw("Failed query to feeder, retrying...",
+				"req", req.URL.String(),
+				"retryAfter", wait.String(),
+				"err", err,
+			)
```

Comment:
This should be debug only when it is on the fast growing stage, after, it has to be a Warning so everyone is aware